### ISSUE TITLE
Layer metadata is no longer available

### ISSUE THREAD

**Issue by nicotejera** [mapbox/mapbox-gl-js#9250]
**1.7.0**:

**Chrome**:

### Steps to Trigger Behavior

 1. Create map
 2. Add custom style with grouped layers

### Link to Demonstration

https://jsbin.com/yepunoc/2/edit?html,js,output

### Expected Behavior

Each layer in the style has `metadata` that includes the `mapbox:group`

### Actual Behavior

The `metadata` is undefined for all layers

**Comment by arindam1993** [comment]
Hi @nicotejera,
This isn't an issue with GL-JS, and is related with how Mapbox Studio serve's styles. 
The `metadata` field from generated by Studio is undocumented and not a part of our API spec, and recently we decided to start shaving it off to reduce the payload size.
However if there's something in there that you're relying on you add the `?metadata=true` as a query param to your url which will force studio to serve it.

Closing since this isn't GL-JS related.
